CHANGELOG.md

```diff
@@ -1,5 +1,55 @@
 - Use Ansible 2.3.1.0 so that we can do shallow clones of tags.
 
+- Role: edx_django_service
```

Comment:
There's only one entry in the `CHANGELOG.md` from the original pull request: 

https://github.com/edx/configuration/pull/4778/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1e

Could you please keep:

```
- Role: edxapp
  - `EDXAPP_EDXAPP_SECRET_KEY` no longer has a default value
```

While removing the others?

Reply:
fixed